core-services/document-grounding/src/main/java/com/sap/ai/sdk/grounding/model/RetrievalSearchResults.java

```diff
@@ -32,7 +32,7 @@ public class RetrievalSearchResults
 // CHECKSTYLE:ON
 {
   @JsonProperty("results")
-  private List<ResultsInner1> results = new ArrayList<>();
+  private List<ResultsInner2> results = new ArrayList<>();
```

Comment:
(Comment)

Okay we need a BLI, one way to fix this is in pre-processing with adding `title: FooBar` to component schema